pkgs/tools/misc/pridefetch/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchFromGitHub, lib, python39, zip }: let
+  version = "1.0.0";
+  sha256 = "sha256-/o4er8bO/3HUFXzP+sC+5DYv9EwmxW05o1RT5fEulEg=";
+
+  name = "pridefetch";
+  src = fetchFromGitHub {
+    owner = "SpyHoodle";
+    repo = name;
+    rev = "v" + version;
+    inherit sha256;
+  };
+in stdenv.mkDerivation {
+  inherit name version;
+  buildInputs = [
+    (python39.withPackages (pythonPackages: with pythonPackages; [
+      distro
+    ]))
+    zip
+  ];
+  dontUnpack = true;
```

Comment:
This doesn't seem necessary: remove this line, add `src` to the derivation, and just do `cd src`.